Cockpit 217 (#312)
cockpit-project/cockpit-project.github.io: _posts/2020-04-15-cockpit-217.md

---
title: Cockpit 217
author: mvo
date: 2020-04-15
tags: cockpit linux
slug: cockpit-217
category: release

> summary: Cockpit with improved ...

Reviewer: TODO :)

comments: 'true'
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 217.

> ### More details about Insights rule hits

Reviewer: Please prefix with the page, "Overview:".

Author: Done.

If your machine is connected to Red Hat Insights, the Health card on
the Overview page will now show more details.

![Insights details](/images/health-insights.png)

### Buttons are now left aligned

In accordance with Patternfly (XXX-LINK-XXX), the buttons in all modal

> dialogs have been left aligned.

Reviewer: And more importantly, Cancel and other secondary buttons have been de-emphasized to be link buttons. (That's not release note ready text, of course).

Author: Fixed.

![New button order](/images/new-button-order.png)

> ### Sending keys to VMs

Reviewer: Please prefix with "Machines:". Also, the general feature has existed for a while, so maybe "Send VT switching keys to VMs"?

Author: Ok! Fixed.

> It is now possible to use a menu to send a number of special keys to a

Reviewer: ... send keys Ctrl+Alt+F1 to F12 to...

Author: Fixed.

---
title: Cockpit 217
author: mvo
date: 2020-04-15
tags: cockpit linux
slug: cockpit-217
category: release
summary: Cockpit with improved Insights details, dialog buttons, and VM consoles
comments: 'true'
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 217.

### Overview: More details about Insights rule hits

> If your machine is connected to Red Hat Insights, the Health card on
> the Overview page will now show more details.

Reviewer 2: ```suggestion
When a machine is connected to Red Hat Insights, the Health card on
the Overview page now shows more details.
```

![Insights details](/images/health-insights.png)

### Dialog buttons have been improved

Following the lead of

> [Patternfly](https://blog.patternfly.org/patternfly/button-placement-on-forms/),

Reviewer: It's spelled "PatternFly".

Author: Fixed.

the buttons in all modal dialogs have been changed to improve

> useability and accessibility.

Reviewer: I think it's "usability".

Author: Fixed.

> Following the lead of
> [PatternFly](https://blog.patternfly.org/patternfly/button-placement-on-forms/),
> the buttons in all modal dialogs have been changed to improve
> usability and accessibility.

Reviewer 2: ```suggestion
Following [PatternFly's revised design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/buttons-and-links#button-placement), button placement in all modal dialogs has been changed [to improve usability and accessibility](https://blog.patternfly.org/patternfly/button-placement-on-forms/).
```

![New button order](/images/new-button-order.png)

### Machines: Sending more keys to the console

> It is now possible to send a number of additional special keys to a
> virtual machine console, for switching Linux virtual consoles.

Reviewer 2: ```suggestion
Many more special keys can be sent to running virtual machines:

- Control+Alt+Delete is usually used for rebooting a computer. On Windows systems, it is also sometimes used for signing in.
- Control+Alt+Backspace is useful for force-quitting the X Window System.
- The rest of of these key combinations are for switching Linux virtual consoles.
```
